### ISSUE TITLE
API - get balance from address

### ISSUE THREAD

**Issue by oopsmonk** [iotaledger/iota.c#48]
Communicates with node API: `GET /api/v1/addresses/{address}`

http status code 200
```
{
  "data": {
    "address": "7ed3d67fc7b619e72e588f51fef2379e43e6e9a856635843b3f29aa3a3f1f006",
    "maxResults": 1000,
    "count": 25,
    "balance": 1338263
  }
}
```

http status code  400
```
{
  "error": {
    "code": "invalid_data",
    "message": "invalid data provided"
  }
}
```

http status code  404
```
{
  "error": {
    "code": "not_found",
    "message": "could not find data"
  }
}
```

**Comment by bernardoaraujor** [comment]
@oopsmonk can you assign this one to me please?

**Comment by bernardoaraujor** [comment]
@oopsmonk what does `v1` mean inside `client/api`?

edit: nevermind, already found [the answer](https://github.com/gohornet/hornet/tree/chrysalis-pt2/plugins/restapi/v1)

**Comment by bernardoaraujor** [comment]
partially blocked by https://github.com/iotaledger/iota.c/issues/59

**Comment by oopsmonk** [comment]
closed in #71
